Task 2/2.1/2.1.2/Polygons.cs

```diff
@@ -0,0 +1,139 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace _2._1._2
+{
+    public abstract class Polygon : Figure //многоугольник
+    {
+        public Point[] points { private set; get; }
```

Comment:
It's common to write getter first, then setter. Difference is one small letter that can lead to confusion and even bugs because of negligence